index.html

```diff
@@ -28,13 +33,150 @@ <h1>I'm Chime Glad to see you!</h1>
         <a href="#">LET'S CONNECT</a>
         <br>
         <br>
-        <ul id="ul">
+        <ul class="ul">
             <li><a href="#"><img src="media/twitter.svg" alt="twitter"></a></li>
             <li><a href="#"><img src="media/Linkedin.svg" alt="Linkedin"></a></li>
             <li><a href="#"><img src="media/M.svg" alt="icon"></a></li>
             <li><a href="#"><img src="media/github.svg" alt="github"></a></li>
             <li><a href="#"><img src="media/hand.svg" alt="icon"></a></li>
         </ul>
-    </section>
+    </div>
+</div>
+
+    <div id="projects-section">
+        <div class="projects">
+            <div class="header">
+            <img src="media/picture.png" alt="media-picture">
+            </div>
+            <div class="content">
+            <h1>Topic</h1>
```

Comment:
- [ ] It's crucial to follow semantics and best practices to improve our projects as much as we can! The `h1` is used as the main title of a website because represents the main heading/subject for the whole page so just used it once throughout your website😉 You could use from **h2 to h6** tags in a hierarchical manner with the rest of your texts 📝